samples/AnomalyDetector/src/Generated/tspCodeModel.json

```diff
@@ -77,7 +77,8 @@
    ],
    "IsExtensible": false,
    "IsNullable": false,
-   "Usage": "Input"
+   "Usage": "Input",
+   "IsLowConfident": false
```

Comment:
I am wondering whether we could use enum instead and call it ConfidentLevel.
Then we could support more values in the future.

Reply:
I changed it to a boolean IsConfident since we only have two values to control whether we will have convenience method or not.